src/axom/sidre/core/Group.hpp

```diff
@@ -640,7 +668,36 @@ class Group
                     TypeID type,
                     int ndims,
                     IndexType* shape,
+                    void* external_ptr )
+  {
+    return createView(path, type, ndims, shape, nullptr, external_ptr);
+  }
+
+  /*!
+   * \brief Create View object with given name or path in this Group that
+   * has a data description with data type and shape and attach
+   * externally-owned data to it.
+   *
+   * If given data type is undefined, or given number of dimensions is < 0,
+   * or given shape ptr is null, method is a no-op.
+   *
+   * This method is equivalent to:
```

Comment:
I think this comment needs to be updated to include a permutation parameter in the "equivalent" code